content/en/docs/Endpoints/getTranscodeStream.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "getTranscodeStream"
+linkTitle: "getTranscodeStream"
+categories:
+- Transcoding
+description: >
+  Returns a transcoded media stream.
+---
+
+`http://your-server/rest/getTranscodeStream`
+
+Returns a transcoded media stream. **Note:** Clients should not try to reconstruct the `transcodeParams`. Instead, they must use the `transcodeParams` provided in the response of the [`getTranscodeDecision`](../gettranscodedecision) endpoint.
+
+### Parameters
+
+| Parameter | Req. | OpenS. | Default | Comment |
+| --- | --- | --- | --- | --- |
+| `songId` | **Yes** | | | The ID of the track to transcode. |
+| `offset` | No | | 0 | The time offset in seconds from which to start transcoding. |
+| `transcodeParams` | **Yes** | | | Server-specific transcoding parameters, obtained from the property `transcodeParams` of the result of the endpoint `getTranscodeDecision`. The value is unique and already properly escaped. |
+
+### Example
+
+{{< alert color="primary" >}} `http://your-server/rest/getTranscodeStream.view?songId=123&offset=0&parameters=...&u=demo&p=demo&v=1.16.1&c=AwesomeClientName&f=json` {{< /alert >}}
+
+### Result
+
+The raw transcoded media stream. In case of an error, a standard HTTP error code is returned with a descriptive message.
```

Comment:
For `/stream.md`, it's explicitly stated that this is an XML document (the standard error). Is this still the case here?